pylutron/__init__.py

```diff
@@ -10,11 +10,13 @@
 from datetime import timedelta
 from enum import Enum
 import logging
+import re
 import socket
 import telnetlib
 import threading
 import time
 
+from slugify import slugify
```

Comment:
This requires updating setup.py so this new depeendency shows up no?

Although I'm still trying to understand if this is really needed.

Reply:
@cdheiser its use isn't extensive so I could to the same thing with concatenation if that makes it easier.  it's not super critical, just let me know.